.azure-devops/full-pipeline.yml

```diff
@@ -12,3 +12,21 @@ jobs:
       - template: ./steps/devcerts.yml
       - template: ./steps/edgewebview.yml
       - template: ./steps/test.yml
+
+  - job: Mac
+    pool:
+      name: OPX Mac Dogfood Pool
+    steps:
+    - template: ./steps/npm-install.yaml
+    - template: ./steps/build.yaml
+    - template: ./steps/run-tests.yaml
+    - template: ./steps/cleanup.yaml
+
+  - job: Linux
+    pool:
+      name: Azure Pipelines
+      vmImage: ubuntu-latest
+    steps:
+    - template: ./steps/npm-install.yaml
+    - template: ./steps/build.yaml
+    - template: ./steps/run-tests.yaml
```

Comment:
The Mac and Linux jobs are using a different test template. The Windows is using `./steps/test.yml`. Why is that different?

Reply:
Good catch.  'run-tests' was the old way and uses npm to trigger, but 'test' uses a command so the logs are live instead of only after the command is finished.  Should be using 'test'.